### PR TITLE
Add traffic driver to new support landing page as discrete test

### PR DESCRIPTION
**Pull request by JustinPinner**
## What does this change?
Sets up an AB test, to provide audience segmentation only, to send traffic to the new support landing page from the epic.

There will be free audience capacity for this from Monday April 3rd as another epic test is coming to an end, so we can do this as a stand-alone (as far as the plumbing for the epic tests will allow) feature instead of trying to bolt it on to an already-in-progress test.

We aren't looking for statistical significance in terms of numbers sent to the landing page, but we will be monitoring the effectiveness of that page in terms of conversions.

## What is the value of this and can you measure success?
We need to drive traffic to the new supporter's landing page that is currently in development (and due for release within the next couple of days), and selected readers who see the epic are potentially a good source for the purpose.

We're going to measure the effectiveness of the page prior to full roll-out.

## Does this affect other platforms - Amp, Apps, etc?
No, web only.

## Screenshots
screen shot 2017-03-29 at 16 28 13

## Tested in CODE?
Not yet.

cc @alexduf @desbo @jranks123 @Ap0c

### CODE REVIEW

**Comment by jranks123** [comment]
My suggestion would be to add the paypal and credit card logo to the element, we have tests that show that it increases the number of overall conversions. See the paypal and credit card variant here https://github.com/guardian/frontend/pull/16241

Will help the test run quicker.

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+        audienceCriteria: 'UK edition readers',
+        audience: 0.2,
+        audienceOffset: 0,
+        showForSensitive: true,
+        useTargetingTool: false,
+
+        overrideCanRun: true,
+        canRun: function () {
+            return config.page.edition.toUpperCase() === 'UK';
```
Comment (jranks123) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: Haven't got time to check now but I thought this was GB...you may be right though

**Comment by jranks123** [comment]
Looks good to me. I would ask that your new page passes on the REFPVID url parameter to the contributions page.

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+        author: 'Justin Pinner',
+        description: 'Use AB framework to push traffic to new supporter landing page.',
+        successMeasure: 'There is no noticeable drop-off in contributions.',
+        idealOutcome: 'Readers sign up for recurring monthly contributions in droves',
+
+        audienceCriteria: 'UK edition readers',
+        audience: 0.2,
+        audienceOffset: 0,
+        showForSensitive: true,
```
Comment (jranks123) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: This test needs to go somewhere between 0.5 and 0.9

**Comment by jranks123** [comment]
You need to add the test to acquisition-test-selector.js and ab-test-clash.js. Apart from that, LGTM

**Review by jranks123 — COMMENTED** [review]
You need to add the test to acquisition-test-selector.js and ab-test-clash.js.

Also, you should use the current control, which has the highlighted sentence (ask one of our devs tomorrow).

As suggested before, I would suggest that you also add in the paypal and credit card icons, as by saying "support the guardian", the CTA is probably a bit weaker and less obviously asking for money (which we know leads to less conversion). I would talk to someone on our team about this. 

Apart from that, and the other comments above, looks good!

**Comment by JustinPinner** [comment]
PayPal is not universally available on all support routes, so we'll leave that off for the time being. Better to not mislead people.

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js (hunk 2 above)
Comment (JustinPinner) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: Will push the offset to 0.5

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js (hunk 1 above)
Comment (alexduf) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: Edition is UK I believe

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    contributionsUtilities,
+    ajax,
+    config,
+    cookies,
+    store,
+    template,
+    contributionsEpicSingleButton
+) {
+
+    function canBeDisplayed() {
```
Comment (alexduf) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: I think this logic doesn't need to be here, the default behaviour of contributionsUtilities is to do what you have here.

Comment (desbo) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: You can get rid of this as the only difference to the default checks in  `contributions-utilities` is the sensitive check, which you can control with `showForSensitive` in the constructor object.

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js (hunk 2 above)
Comment (alexduf) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: This needs to be defined on Monday

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        successMeasure: 'There is no noticeable drop-off in contributions.',
+        idealOutcome: 'Readers sign up for recurring monthly contributions in droves',
+
+        audienceCriteria: 'UK edition readers',
+        audience: 0.2,
+        audienceOffset: 0,
+        showForSensitive: true,
+        useTargetingTool: false,
+
+        overrideCanRun: true,
```
Comment (alexduf) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: This needs to be false in order to inherit from all the default behaviour. The canRun function bellow will still be taken into account. (see contributions-utilities.js:100)

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                        p1: '… we have a small favour to ask. More people are reading the Guardian than ever but far fewer are paying for it. And advertising revenues across the media are falling fast. So you can see why we need to ask for your help. The Guardian\'s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.',
+                        p2: 'If everyone who reads our reporting, who likes it, helps to pay for it, our future would be much more secure.',
+                        p3: '',
+                        cta1: 'Support the Guardian'
+                    });
+                },
+
+                insertBeforeSelector: '.submeta',
+
+                test: function (render) {
```
Comment (alexduf) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: that `test` function can be removed and will rely on the default behaviour as well

File: static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+                    if (canBeDisplayed()) render();
+                },
+                isUnlimited : true,
```
Comment (alexduf) [inline static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-to-support-landing-page.js]: this should be removed

**Review by alexduf — COMMENTED** [review]
as we chatted about I think it would convert better by using our latest control (highlight)

**Review by desbo — APPROVED** [review]
Nice

**Comment by JustinPinner** [comment]
Here's the latest screen capture (I meant to attach this with the earlier commits);
screen shot 2017-03-30 at 11 22 09

**Pull request merged**